src/figuresClasses.ts

```diff
@@ -2,10 +1,89 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+function getNumberWithTwoDecimalPlaces(number: number): number {
+  return Math.floor(number * 100) / 100;
+}
+
+export interface Figure {
+  shape: Shape;
+  color: Color;
+
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  readonly shape: Shape = 'triangle';
 
-export class Circle implements Figure {}
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Side cannot be 0 and less');
```

Comment:
[CODE STYLE] - The error message should be more descriptive to better indicate what went wrong. Consider providing a message that indicates that triangle sides must be positive numbers.